### PR TITLE
Bahrain and South Africa Regions Release Notes

### PR DESCRIPTION
**Pull request by MarkvanMents**
(Changes also made to Evaluation Guide: https://www.mendix.com/evaluation-guide/app-lifecycle/mendix-cloud-overview/]

### CODE REVIEW

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/mendix-ip-addresses.md [hunk 1]
```diff
@@ -33,16 +33,20 @@ Mendix uses the following static IP addresses. If you need further information,
...
+{{% todo %}}Add IP Addresses{{% /todo %}}
+
 | Region | Location | IP Addresses |
 | --- | --- | ---|
 | Mendix Cloud V4 Free Tier EU | Ireland (eu-west-1) | `52.211.106.230` <br /> `52.18.20.119` <br /> `52.18.122.238` |
 | Mendix Cloud V4 Asia Pacific | Singapore (ap-southeast-1) | `52.77.0.52` <br /> `122.248.194.24` <br /> `54.179.212.86` |
 | Mendix Cloud V4 Asia Pacific | Sydney (ap-southeast-2) | `13.54.220.143` <br /> `13.54.176.58` <br /> `13.238.3.124` |
 | Mendix Cloud V4 Asia Pacific | Tokyo (ap-northeast-1) |  `52.198.238.110` <br /> `13.113.58.15` <br /> `52.193.228.1` |
 | Mendix Cloud V4 Asia Pacific | Mumbai (ap-south-1) |  `65.1.183.103` <br /> `43.205.31.32` <br /> `43.204.254.95` |
+| Mendix Cloud V4 Bahrain | Bahrain (me-south-1) | |
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/mendix-cloud-deploy/mendix-ip-addresses.md]: IP Addresses

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/mendix-ip-addresses.md [hunk 2]
```diff
@@ -33,16 +33,20 @@ Mendix uses the following static IP addresses. If you need further information,
...
 | Mendix Cloud V4 Canada | Canada (ca-central-1) | `3.97.111.124` <br /> `3.98.194.54` <br /> `35.182.171.81` |
 | Mendix Cloud V4 EU  | Frankfurt (eu-central-1) | `52.58.42.15` <br /> `35.157.102.14` <br /> `35.156.112.28` |
 | Mendix Cloud V4 IE | Ireland (eu-west-1) | `52.18.10.62` <br /> `52.208.39.99` <br /> `99.80.177.69` |
+| Mendix Cloud V4 South Africa | Cape Town (af-south-1) | |
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/mendix-cloud-deploy/mendix-ip-addresses.md]: IP Addresses

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/mxcloudv4/_index.md [hunk 3]
```diff
@@ -33,6 +33,8 @@ The primary hosting locations are as follows:
 * Mendix Cloud Canada: AWS Canada
 * Mendix Cloud EU: AWS Dublin
 * Mendix Cloud EU: AWS Frankfurt
+* Mendix Cloud Middle East: AWS Bahrain
+* Mendix Cloud South Africa: AWS Cape Town
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/mendix-cloud-deploy/mxcloudv4/_index.md]: What are the actual names of these going to be?
AWS uses: Middle East and Africa
We use: Bahrain and South Africa
Data centers are: Bahrain and Cape Town

File: content/en/docs/developerportal/operate/backups/_index.md [hunk 4]
```diff
@@ -29,17 +29,21 @@ The following backups are retained:
 
 The **Start Time** of nightly backups in UTC is shown below. The **Local Time** indicates the time at the regional data center – this may vary if *Summer Time* or other adjustments are in place. The **Estimated Duration** indicates the period during which backups are expected to be taken from apps in that region – your app can be backed up at any time during this period and the exact period during which backups are taken can change due to factors at the data center which are outside Mendix's control.
 
+{{% todo %}}Add rest of backup information{{% /todo %}}
+
 | Region | Replication Region | Start Time (UTC) | Local Time | Estimated Duration |
 | --- | --- | --- | --- |  --- |
+| Bahrain | Mumbai | | | |
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/operate/backups/_index.md]: Start time, Local time, Duration

File: content/en/docs/developerportal/operate/backups/_index.md [hunk 5]
```diff
@@ -29,17 +29,21 @@ The following backups are retained:
...
+| Canada | Oregon | 05:00 | 00:00 | 1 hour |
 | Dublin | Frankfurt | 23:00 | 00:00 |  3 hours |
 | Frankfurt | Dublin | 00:00 | 01:00 | 3 hours |
 | London | Frankfurt | 23:00 | 23:00 | 2 hours |
+| Mumbai | Singapore | 19:30 | 01:00 | 1 hour |
 | Oregon | Ohio | 07:00 | 00:00 | 1 hour |
 | Singapore | Sydney | 17:00 | 01:00 | 1 hour |
+| Cape Town | Frankfurt | | | |
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/operate/backups/_index.md]: Start time, Local time, Duration

File: content/en/docs/releasenotes/deployment/mendix-cloud.md [hunk 6]
```diff
@@ -17,7 +17,13 @@ For information on the current status of deployment to Mendix Cloud and any plan
 
 ## 2023
 
-### January 5th, 2022
+### January 13th, 2023
+
+#### Improvements
+
+* We added the AWS Bahrain and AWS South Africa regions as a new deployment targets.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/deployment/mendix-cloud.md]: The AWS regions are actually Bahrain and Cape Town, not South Africa.

**Review by MarkvanMents — COMMENTED** [review]
Outstanding questions/clarifications

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/mxcloudv4/_index.md (hunk 3 above)
Comment (MarkvanMents) [reply content/en/docs/developerportal/deploy/mendix-cloud-deploy/mxcloudv4/_index.md]: Aligned with Status page and AWS nomenclature

File: content/en/docs/releasenotes/deployment/mendix-cloud.md (hunk 6 above)
Comment (MarkvanMents) [reply content/en/docs/releasenotes/deployment/mendix-cloud.md]: Aligned with AWS nomenclature

**Comment by Sakshi-Dhakad** [comment]
Looks good to me so far!  Let me know Mark if you need any further information before releasing this

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/mendix-ip-addresses.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/developerportal/deploy/mendix-cloud-deploy/mendix-ip-addresses.md]: Added

**Comment by Sakshi-Dhakad** [comment]
Reviewed and good to go, thanks @MarkvanMents for picking this up urgently! 👍

**Pull request merged**